iungo-web/img-compressor-cli/0.1/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+	"copy-from-package": {
+		"bin/compressor": ""
+	},
+	"env": {
+		"API_KEY": "123ABC1",
+		"DIRETORIO_RAIZ": "seu/diretorio/aqui/"
+	},
```

Comment:
I dont see that your recipe is using these.

Reply:
The recipe does not use it, it will only create these lines in the .env file for the compression system to use. 

[https://github.com/iungoweb/img-compressor-cli/blob/main/src/Compressor.php#L49](https://github.com/iungoweb/img-compressor-cli/blob/main/src/Compressor.php#L49)

Reply:
Sure, lets keep them.